src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.input__big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.input__small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.input__big::placeholder,
+.input__small::placeholder {
+  color: #3d4e61;
+}
+
+.input {
+  box-sizing: border-box;
+  font-family: inherit;
+  width: 100%;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.icon {
+  position: absolute;
+}
+
+.icon__big {
+  top: 25px;
+  left: 26px;
+}
+
+.icon__small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  outline: none;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
```

Comment:
placeholder should not change font-weight on focus, only input text
<img width="308" alt="image" src="https://user-images.githubusercontent.com/66749919/198828277-85ec0fde-3220-4e9c-a8bd-141a95e13ea4.png">

Reply:
I don't understand how to do it!